nanovdb/nanovdb/tools/GridBuilder.h

```diff
@@ -1968,7 +1968,7 @@ void Grid<BuildT>::operator()(const Func& func, const CoordBBox& bbox, ValueType
                 NANOVDB_ASSERT(leaf == nullptr);
             }
         }// loop over sub-part of leafBBox
-        if (leaf) delete leaf;
```

Comment:
Ah, the ghost of past compilers continue to haunt us.

Thankfully modern compilers and language specs have unified so we can reliably not test for null before deleting.